### ISSUE TITLE
OCX compiles / runs correctly, however Mouse events unrecognized in IDE

### ISSUE THREAD

**Issue by LastJasonNext** [twinbasic/twinbasic#1605]
**Describe the bug**
A tree view OCX (ccrpftv.ocx 1999) builds and runs correctly, IDE fails to recognize OCX mouse events

**To Reproduce**
1. import VBP
2. save as twinproject
3. Build
4. Launch EXE
5. Project operates correctly
6. Close, reload from twinproj
7. Play project (run)
8. ccrpftv.ocx appears correctly, use mouse click
9. OCX SelectChange event fails to fire
10. Stop, double-click on form 's tree view OCX
11. Access to event [name]_Click instead of desired [name]_SelectChange
12. OCX graphic disappears from form
 
**Expected behavior**
 [name]_SelectChange event expected to fire during IDE Run

**Screenshots**
1. import, save to twinproj, build, Launch EXE
![image](https://github.com/twinbasic/twinbasic/assets/130419891/0c531263-1c3e-4525-bbab-0c9c7737d701)
![image](https://github.com/twinbasic/twinbasic/assets/130419891/e32581f7-4881-44c3-b185-59b9a0ee25c9)
2. load twinproj, Run in IDE, double click on ccrpftv.ocx, should event to here:
![image](https://github.com/twinbasic/twinbasic/assets/130419891/ac437fe7-0eaf-4ae6-b0c4-37c99bed0542)
3. instead events to here:
![image](https://github.com/twinbasic/twinbasic/assets/130419891/d296485a-ce4d-4c8a-bfb2-873877540e85)
4, OCX graphic clears:
![image](https://github.com/twinbasic/twinbasic/assets/130419891/cfff0e77-caae-485f-b5f1-ce70850ffd40)

**Desktop (please complete the following information):**
 - OS: latest Windows 11 (as of 23.07.05)
 - twinBASIC compiler version IDE_BETA_346

**Additional context**
The name of this Tree View OCX I found way back in the day (1999?), ccrpftv.ocx, pretty much discloses the original source. I can upload it to you if interested, however there may be a similar file-oriented tree view OCX out...there...somewhere that TB may have better luck with. If you know of one? I'll check with social. Perhaps this one is feature rich enough to get passed around and worth TB IDE adaptation. Another solution is, of course, to build a file-oriented usercontrol using VB6 CommonControls TreeView OCX. That'd be a nice TB addition, perhaps already in the works for your project. That really was an obvious, glaring omission in VB6. Thanks!

**Comment by fafalone** [comment]
Yes the source code would be extremely useful in diagnosing bugs.

As for alternatives, I don't recall testing all the mouse event specifically (besides click/right click to navigate), but I have ported my [ucShellTree control](https://github.com/fafalone/ShellControls)  to tB, made it 64bit compatible, and gotten it running in VB6, 64bit VBA, and .NET, seemingly without any issues.

![image](https://github.com/twinbasic/twinbasic/assets/7834493/661cf529-536f-4ff7-af2f-c0dfcbb54acf)

It also seamlessly integrates with ucShellBrowse, the other side of the tree, which is a separate control (but [in the same repo](https://github.com/fafalone/ShellControls)):

![image](https://github.com/twinbasic/twinbasic/assets/7834493/e79d1b28-0c0b-47b3-bc22-64f17dbf9a3c)
